spec/ndx-photometry.extensions.yaml

```diff
@@ -143,6 +146,7 @@ groups:
   - name: frequency
     dtype: float
     doc: voltage frequency in unit hertz
+    quantity: '?'
```

Comment:
I left this optional for now, but feels a duplication with `rate` of `TimeSeries`.

Reply:
I think this might be referring to the frequency of the alternating current, e.g. 60 Hz in the US and 50 Hz in Europe

Reply:
but maybe not, I don't really remember

Reply:
I see, is it okay to leave it optional or rollback?